runtime/common/src/lib.rs

```diff
@@ -22,7 +22,7 @@ pub mod weights;
 #[macro_export]
 macro_rules! prod_or_testnet_or_local {
 	($prod:expr, $test:expr, $local:expr) => {
-		if cfg!(any(feature = "frequency-local", feature = "frequency-no-relay")) {
+		if cfg!(any(feature = "frequency-local", feature = "frequency-no-relay", feature = "frequency-bridging")) {
```

Comment:
This should probably be `frequency-westend-local`.

Reply:
Create issue to revisit properly feature flagging with `frequency-westend-local` and frequency-westend-testnet.